<launch>

  <node pkg="nodelet" type="nodelet" name="nodelet_manager"  args="manager" output="screen"/>

  <node pkg="nodelet" type="nodelet" name="feature_extraction" args="load extrinsic_calibration/feature_extraction nodelet_manager" output="screen" required="true">
    <param name="plane_distance_threshold" value="0.08"/>
    <param name="line_distance_threshold" value="0.08"/>
  </node>

  <node pkg="cam_lidar_calibration" type="input_sample" name="input_sample" output="screen">
  </node>
  
  <node pkg="cam_lidar_calibration" type="optimizer" name="optimizer" output="screen">
  </node>
  
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find cam_lidar_calibration)/rviz/cam_lidar_calibration.rviz" />

  <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" />

</launch>
